priv/repo/migrations/20160805203929_create_skill.exs

```diff
@@ -0,0 +1,16 @@
+defmodule CodeCorps.Repo.Migrations.CreateSkill do
+  use Ecto.Migration
+
+  def change do
+    create table(:skills) do
+      add :title, :string
+      add :description, :string
+      add :original_row, :integer
+      add :slug, :string
+
+      timestamps()
+    end
+
+    create index(:skills, [:slug], name: :index_skills_on_slug, unique: true)
```

Comment:
There should never be an issue with that because of the Inflex dep `parameterize` method that creates the slug will always be lowercase